spec/helpers/measures_helper_spec.rb

```diff
@@ -44,4 +44,24 @@
       it { expect(helper.legal_act_regulation_url_link_for(measure)).to be_html_safe }
     end
   end
+
+  describe 'check_how_to_export_goods_link' do
+    subject(:link) { check_how_to_export_goods_link(commodity_code: commodity_code, country_code: '', country_name: '') }
```

Comment:
Foregoing the helper has weird behaviour with assigns in my experience of writing helper specs

```suggestion
    subject(:link) { helper.check_how_to_export_goods_link(commodity_code: commodity_code, country_code: '', country_name: '') }
```